admin/lib/gaku/admin.rb

```diff
@@ -10,7 +10,7 @@
 require 'momentjs-rails'
 require 'underscore-rails'
 require 'nprogress-rails'
-require 'rails4_client_side_validations'
+require 'client_side_validations'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.